drivers/espi/espi_mchp_xec_v2.c

```diff
@@ -852,12 +852,16 @@ static void espi_pc_isr(const struct device *dev)
 	struct espi_iom_regs *regs = ESPI_XEC_REG_BASE(dev);
 	uint32_t status = regs->PCSTS;
 
+	LOG_DBG("%s %x", __func__, status);
 	if (status & MCHP_ESPI_PC_STS_EN_CHG) {
 		if (status & MCHP_ESPI_PC_STS_EN) {
 			setup_espi_io_config(dev, MCHP_ESPI_IOBAR_INIT_DFLT);
 		}
 
 		regs->PCSTS = MCHP_ESPI_PC_STS_EN_CHG;
+	} else if (status & MCHP_ESPI_PC_STS_BUS_ERR) {
```

Comment:
why `else if`? can we have only `if` for the bus error check? if so, we could move this check first in the function.

Reply:
Yes, we can have multiple bits set, corrected.